### PR TITLE
Prepare release 2.0.0a3

### PR DESCRIPTION
**Pull request by mprazz**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -167,7 +167,7 @@ optional = true
 
 [tool.poetry.dependencies.tensorflow-text]
 version = "~2.3"
-markers = "sys_platform!='win32'"  # TODO: remove this marker once tensorflow-text supports windows
+markers = "sys_platform!='win32'"
```
Comment (wochinge) [inline pyproject.toml:170]: can we please re-add this comment?

**Review by wochinge — APPROVED** [review]
🚀

File: pyproject.toml (hunk 1 above)
Comment (mprazz) [inline pyproject.toml:170]: This `TODO` was removed automatically it seems. Should it be added back?

Comment (mprazz) [reply pyproject.toml:170]: LOL same thought!

Comment (mprazz) [reply pyproject.toml]: Done! 😊

**Review by m-vdb — APPROVED** [review]

**Pull request merged**